src/services/admission/admission-formats.ts

```diff
@@ -0,0 +1,28 @@
+export interface DecisionInfoEntry {
+    _id?: string;
+    userId: string;
+    name: string;
+    status: DecisionStatus;
+}
```

Comment:
Where is this being used? Can we use the DecisionInfo class from `database/decision` instead?

Reply:
wait nvm - correct me if i'm wrong, but this is the input that we're getting from the admin site. keep the interface, but can you rename it to `ApplicantDecisionFormat` or something along those lines?